src/App.jsx

```diff
@@ -1,80 +1,17 @@
 import React from 'react';
+
+import Welcome from './components/Welcome/Welcome';
+import Header from './components/Header/Header';
+import Article from './components/Article/Article';
```

Comment:
if you create an index file for each component - the import would be bit shorter:
```suggestion
import Welcome from './components/Welcome';
import Header from './components/Header';
import Article from './components/Article';
```